utils/featuresConfigUtils.ts

```diff
@@ -0,0 +1,60 @@
+import Stripe from "stripe";
+
+export const defaultFeaturesConfig = {
+  payments: {
+    refund_management: true,
+    dispute_management: true,
+    capture_payments: true,
+  },
+  payment_details: {
+    refund_management: true,
+    dispute_management: true,
+    capture_payments: true,
+  },
+};
+
+export type FeaturesConfig = typeof defaultFeaturesConfig;
+
+export const featuresCache: {
+  [accountId: string]: { featuresConfig: FeaturesConfig; updated: boolean };
+} = {};
+
+export type Component = keyof typeof defaultFeaturesConfig;
+export type Feature = keyof typeof defaultFeaturesConfig[Component];
+
+export function accountSessionFeaturesUpToDate(accountId: string) {
```

Comment:
If we go with the local storage suggestion, I think most of this file can be deleted